### ISSUE TITLE
Title Slide Scriptures

### ISSUE THREAD

**Issue by alanmsings** [hymns-for-worship/issues#142]
Would you consider using a modern Bible version for the Title Slide Scriptures?

**Comment by rodydavis** [comment]
Are you talking about the title slides?

And which Bible version are you looking for?

The title slides get generated ahead of time, but it might be possible to customize them in the future.

**Comment by alanmsings** [comment]
Yes, sir. I am referring to the Title Slides. The NASB 1995 or ESV would work well for the Title Slides.
